src/common.rs

```diff
@@ -48,9 +47,9 @@ impl FileEntry {
             None => return None,
         };
 
-        let content = match Self::get_content_by_path(&path) {
-            Some(s) => s,
-            None => return None,
+        let content = match fs::read_to_string(&path) {
+            Ok(content) => content,
+            Err(_) => return None,
         };
```

Comment:
```suggestion
        let content = fs::read_to_string(&path)?;
```
Can we use the question mark method instead of `match`?

Reply:
That would mean we need to return some proper kind of `Result/Option`-type which respects errors in that function instead of `Option<(Self, Vec<String>)>` which leads to some further changes.
Alternatively we could extract this line into an own function and act accordingly on the result in the `from` function.

Reply:
Guys, I'm a little late, but actually one could use:
`let content = fs::read_to_string(&path).ok()?;`

ok () converts the `Result` to `Option`.

I apologize, without the @mgrachev comment, I did not pay attention to this.
But I think this is not so important, just for the future :relaxed: